recipes/gainput/all/patches/0002-fix-cmake-win32-condition.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index a443b66..2c920eb 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -9,7 +9,7 @@ option(GAINPUT_TESTS "Build Tests for Gainput" ON)
+ option(GAINPUT_BUILD_SHARED "BUILD_SHARED" ON)
+ option(GAINPUT_BUILD_STATIC "BUILD_STATIC" ON)
+ 
+-if(!WIN32)
++if(NOT WIN32)
```

Comment:
Did this ever work?

Reply:
IMHO it is broken upstream, but I don't see the project live enough to submit the patch (3 years since the last commit).

In a native build, `CMAKE_SYSTEM_VERSION` takes the value of `CMAKE_HOST_SYSTEM_VERSION`, but in cross-building you need to provide the value explicitly and it crashes... but yes, the condition is wrong outside Windows.